edweiss-app/package-lock.json

```diff
@@ -9934,27 +9934,6 @@
       "license": "MIT",
       "optional": true
     },
-    "node_modules/@types/send": {
```

Comment:
Is it normal to have removed this? I also noticed that sometimes in my code even if I didn't do anything to this file, these elements would disappear, but I did not know the reason. Are you aware that this was removed and that it should be indeed removed?

Reply:
Oh, I did not even knew of it existance. Thank's for raising your concern about it.
To be completly honest, I have branch from main and then I `npm install` stuff from `edweiss-app` and `edweiss-firebase/functions`. It may have did it automatically. 

I don't know what it's supposed to do. What are your insight on it ?